index.js

```diff
@@ -155,7 +155,24 @@ ShareDbMongo.prototype._connect = function(mongo, options) {
     mongo(finish);
     return;
   }
-  mongodb.connect(mongo, options, finish);
+  // Get mongo options from the explicitly defined field `mongoOptions` or
+  // sanitize the whole options object by removing all ShareDB-specific options
+  // TODO: Deprecate the ability to pass mongo options implicitly
+  var mongoOptions;
+  if (options.mongoOptions) {
+    mongoOptions = options.mongoOptions;
+  } else {
```

Comment:
I'd remove this part where we delete all the sharedb-mongo options and instead just require people to update their options if they want to use the new driver